proposals/0448-type-variable-scoping.rst

```diff
@@ -551,10 +564,13 @@ Effects
    We expect a future proposal to remedy this problem, with either a modifier or some symbol.
    For example, perhaps we would say e.g. ``f (Just @(*a) x) = ...`` to denote an occurrence of already-in-scope type variable ``a``.
 
-#. Because ``-XScopedTypeVariables`` implies ``-XTypeAbstractions``,
-   people using ``-XScopedTypeVariables`` would have access to the new features without enabling a new extension.
-   This is backward-compatible with the current implementation, which requires both ``-XScopedTypeVariables`` and ``-XTypeApplications`` to be in effect.
-   (With this proposal, ``-XScopedTypeVariables`` alone would be enough.)
+#. Backward-compatibility with the current implementation,
+   which requires both ``-XScopedTypeVariables`` and ``-XTypeApplications`` to be in effect but nothing more,
```

Comment:
What is this "nothign more" bit? I'm lost.

Reply:
```suggestion
   which requires both ``-XScopedTypeVariables`` and ``-XTypeApplications`` to be in effect but does not require any extension dedicated to this feature,
```
just trying to restart the status quo. Here is a longer version, but we could also just do something like
```suggestion
   which merely requires ``-XScopedTypeVariables`` and ``-XTypeApplications`` to be in effect,
```

Reply:
@simonpj does one of these look good?

Reply:
good!